doxygen/Doxyfile

```diff
@@ -865,25 +985,27 @@ EXCLUDE_PATTERNS       =
 # (namespaces, classes, functions, etc.) that should be excluded from the
 # output. The symbol name can be a fully qualified name, a word, or if the
 # wildcard * is used, a substring. Examples: ANamespace, AClass,
-# AClass::ANamespace, ANamespace::*Test
+# ANamespace::AClass, ANamespace::*Test
 #
 # Note that the wildcards are matched against the file with absolute path, so to
 # exclude all test directories use the pattern */test/*
 
-EXCLUDE_SYMBOLS        =
+EXCLUDE_SYMBOLS        = HID_API_AS_STR_IMPL \
+                         HID_API_AS_STR \
+                         HID_API_TO_VERSION_STR
```

Comment:
This is the important part that allows eliding specific members from the generated docs. ✏️ Here, I used it to exclude the "helper macros" that don't have a docstring.